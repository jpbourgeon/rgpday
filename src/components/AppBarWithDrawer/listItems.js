import React from 'react'
import ListItem from '@material-ui/core/ListItem'
import ListItemIcon from '@material-ui/core/ListItemIcon'
import ListItemText from '@material-ui/core/ListItemText'
import ViewListIcon from '@material-ui/icons/ViewList'
import MultilineChartIcon from '@material-ui/icons/MultilineChart'
import PersonIcon from '@material-ui/icons/Person'

export const mainListItems = (
  <div>
    <ListItem button>
      <ListItemIcon>
        <ViewListIcon />
      </ListItemIcon>
      <ListItemText primary='Menu A' />
    </ListItem>
    <ListItem button>
      <ListItemIcon>
        <MultilineChartIcon />
      </ListItemIcon>
      <ListItemText primary='Menu B' />
    </ListItem>
  </div>
)

export const secondaryListItems = (
  <div>
    <ListItem button>
      <ListItemIcon>
        <PersonIcon />
      </ListItemIcon>
      <ListItemText primary='Menu C' />
    </ListItem>
  </div>
)
