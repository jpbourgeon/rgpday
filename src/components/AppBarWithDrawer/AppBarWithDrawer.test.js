/* global describe, it, expect */
import React from 'react'
import renderer from 'react-test-renderer'
import AppBarWithDrawer from './'
import jssSerializer from '../../../utils/jssSnapshotSerializer'

expect.addSnapshotSerializer(jssSerializer)

describe('The AppBarWithDrawer component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<AppBarWithDrawer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
