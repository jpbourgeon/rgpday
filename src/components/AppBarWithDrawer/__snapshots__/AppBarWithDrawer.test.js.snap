// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The AppBarWithDrawer component should render correctly 1`] = `
<div
  className="root-0"
>
  <header
    className="root-20 elevation4-26 root-11 positionFixed-12 colorPrimary-18 mui-fixed appBar-3"
  >
    <div
      className="root-47 regular-49 toolbar-1"
    >
      <button
        aria-label="Open drawer"
        className="root-93 root-87 colorInherit-88 menuButton-4"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onContextMenu={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="0"
        type="button"
      >
        <span
          className="label-92"
        >
          <svg
            aria-hidden="true"
            className="root-96"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
            />
          </svg>
        </span>
        <span
          className="root-145"
        />
      </button>
      <h1
        className="root-51 h6-68 colorInherit-80 noWrap-77 title-5"
      >
        RGPDay
      </h1>
    </div>
  </header>
  <nav
    className="drawer-6"
  >
    <div
      className="root-105 docked-106"
    >
      <div
        className="root-20 elevation0-22 paper-107 drawerPaper-7 paperAnchorLeft-108 paperAnchorDockedLeft-112"
      >
        <div
          className="toolbarIcon-2"
        >
          <button
            className="root-93 root-87"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex="0"
            type="button"
          >
            <span
              className="label-92"
            >
              <svg
                aria-hidden="true"
                className="root-96"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 7.41L14 6l-6 6 6 6 1.41-1.41L10.83 12z"
                />
                <path
                  d="M0 0h24v24H0z"
                  fill="none"
                />
              </svg>
            </span>
            <span
              className="root-145"
            />
          </button>
        </div>
        <hr
          className="root-117"
        />
        <ul
          className="root-122 padding-123"
        >
          <div>
            <div
              className="root-93 root-126 default-129 gutters-134 button-135"
              onBlur={[Function]}
              onContextMenu={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="root-138"
              >
                <svg
                  aria-hidden="true"
                  className="root-96"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M4 14h4v-4H4v4zm0 5h4v-4H4v4zM4 9h4V5H4v4zm5 5h12v-4H9v4zm0 5h12v-4H9v4zM9 5v4h12V5H9z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
              </div>
              <div
                className="root-139"
              >
                <span
                  className="root-51 body1-60 primary-142"
                >
                  Menu A
                </span>
              </div>
              <span
                className="root-145"
              />
            </div>
            <div
              className="root-93 root-126 default-129 gutters-134 button-135"
              onBlur={[Function]}
              onContextMenu={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="root-138"
              >
                <svg
                  aria-hidden="true"
                  className="root-96"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M22 6.92l-1.41-1.41-2.85 3.21C15.68 6.4 12.83 5 9.61 5 6.72 5 4.07 6.16 2 8l1.42 1.42C5.12 7.93 7.27 7 9.61 7c2.74 0 5.09 1.26 6.77 3.24l-2.88 3.24-4-4L2 16.99l1.5 1.5 6-6.01 4 4 4.05-4.55c.75 1.35 1.25 2.9 1.44 4.55H21c-.22-2.3-.95-4.39-2.04-6.14L22 6.92z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
              </div>
              <div
                className="root-139"
              >
                <span
                  className="root-51 body1-60 primary-142"
                >
                  Menu B
                </span>
              </div>
              <span
                className="root-145"
              />
            </div>
          </div>
        </ul>
        <hr
          className="root-117"
        />
        <ul
          className="root-122 padding-123"
        >
          <div>
            <div
              className="root-93 root-126 default-129 gutters-134 button-135"
              onBlur={[Function]}
              onContextMenu={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex="0"
            >
              <div
                className="root-138"
              >
                <svg
                  aria-hidden="true"
                  className="root-96"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
                  />
                  <path
                    d="M0 0h24v24H0z"
                    fill="none"
                  />
                </svg>
              </div>
              <div
                className="root-139"
              >
                <span
                  className="root-51 body1-60 primary-142"
                >
                  Menu C
                </span>
              </div>
              <span
                className="root-145"
              />
            </div>
          </div>
        </ul>
      </div>
    </div>
  </nav>
  <main
    className="content-10"
  >
    <div
      className="appBarSpacer-9"
    />
  </main>
</div>
`;
